escapades_backend/spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { create(:user) }
+  it "is valid with valid attributes" do
+    expect(user).to be_valid
+  end
```

Comment:
I think we could check it better here, using values instead of factories maybe. Or using a valid/invalid trait?

Reply:
I dont know. I left this as a basic test but I think we better not test model because it is rails's core system (unless we want to test custom methods)

Reply:
Ok for the moment there is no validation so let's keep it like this, but let's add more complexity when we add custom validations on attributes.